lib/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
```

Comment:
Good question!

There's no way to automatically check for newer versions.
IEC doesn't have some kind of getLatest API call to get the latest XSD versions. It's always an URL containing the full version, which we have to include in our application.

By doing this, you always have to update your application when there's a newer version (because you have to update the URL). So it can't be done automatically.

So I propose just a library containing the XSD's, and if there is a newer version (which will not be that often, because it takes like 2 years for a new version) is can be done manually. This doesn't have to be checked every week.
I also created a Github Action for packaging it to Github Packages (https://github.com/com-pas/compas-scl-xsd/packages/797160), which will be triggered on every push (this is debatable).

Reply:
I'm agree to build the packages at each push, but I think that we should have a warning if the standard change the content of one existing version (like correcting a tissue) Maybe your are right verify that every week is overkill, but maybe we could verify that once a month.

Reply:
How can we do that without manually changing our code and automate everything?
We need some kind of getLatest and just check the checksum if there are any changes, but IEC doesn't have something like that. The Download URL is always absolute.

How do you see this?

Reply:
In this case we could download compute the checksum, validate with the one that we are stored and raise an error if there is a difference to force human to check why there is a difference and then handle code modification.
And when standard deliver a new edition, we have time to integrate it correctly in our code.

Reply:
The checksum never changes because it's the same URL with the same version in it.
If the checksum would change, the URL also changes because the version is in it. That's the whole problem.
If there is a new version, we can't know before changing the URL manually.

Reply:
The file name contains only the main edition but not if he contains tissue correction.

Reply:
Okay, I didn't know edits doesn't always mean that version gone up.
Last version for example is 2007B4 (https://assets.iec.ch/public/tc57/IEC_61850-6.2018.SCL.2007B4.full.zip), so this version stays the same also after minor edits (which is possible).

I will add the Action again :)
If we think it's still overkill, we can remove it afterwards.

Reply:
Thanks